### PR TITLE
feature. Modify TDataGrid header checkbox position

### PR DESCRIPTION
**Pull request by kkusaeng**
# Key Changes of PR
- Modify TDataGrid header checkbox position

# Screenshots

## before
image

## after 
image

### CODE REVIEW

**Review by Siyeop — APPROVED** [review]
0.5.4 로 반영했습니다.

File: src/styles/component/data-container/drop-holder/TDropHolder.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@
     .t-drop-holder__items {
         position: absolute;
         z-index: $t-z-index-drop-holder;
-        border: 1px solid $t-gray-color-3;
+        border: 1px solid $t-blue-gray-color-3;
```
Comment (Siyeop) [inline src/styles/component/data-container/drop-holder/TDropHolder.scss:17]: 제가 맘대로 이거 하나 넣었어요

**Comment by kkusaeng** [comment]
[surprised]     이준희 reacted to your message:
________________________________

**Comment by kkusaeng** [comment]
[like]  이준희 reacted to your message:
________________________________

**Pull request merged**